### ISSUE TITLE
Caught java.lang.NoSuchFieldError: FACTION_NO_LEADER

### ISSUE THREAD

**Issue by Rsl1122** [Rsl1122/Plan-PlayerAnalytics#232]
```
main.java.com.djrapitops.plan.utilities.analysis.Analysis Caught java.lang.NoSuchFieldError: FACTION_NO_LEADER
com.djrapitops.pluginbridge.plan.factions.FactionsTable.getHtmlReplaceValue(FactionsTable.java:80)
main.java.com.djrapitops.plan.utilities.analysis.Analysis.lambda$analyzeAdditionalPluginData$5(Analysis.java:210)
java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
java.util.stream.AbstractPipeline.copyInto(Unknown Source)
java.util.stream.ForEachOps$ForEachTask.compute(Unknown Source)
java.util.concurrent.CountedCompleter.exec(Unknown Source)
java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
java.util.concurrent.ForkJoinPool$WorkQueue.runTask(Unknown Source)
java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
----
```

**Comment by Rsl1122** [comment]
Moved to LOW priority:
- Analysis Catches NoSuchFieldError properly (Does not prevent analysis from being cached)

**Comment by Rsl1122** [comment]
Apparently reverted somehow

**Comment by Rsl1122** [comment]
Fixed (By reverting revert) a2c4f150c547e21416f74ddf2cf41fbdd871b58a
